docs/INSTALL.md

```diff
@@ -458,6 +458,11 @@ the following:
   to provide a reliable delivery of notifications (e.g.
   `--zmqpubrawblock=tcp://127.0.0.1:28332` and
   `--zmqpubrawtx=tcp://127.0.0.1:28333`).
+- Make sure you do NOT use the bitcoind config setting `--rpcserialversion`.
```

Comment:
```suggestion
- Make sure the config `--rpcserialversion` in `bitcoind` is NOT used. 
```

Reply:
Ah, I missed this before, but the config flags in `bitcoind` only have a single dash.